src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewViewModel.cs

```diff
@@ -0,0 +1,873 @@
+		//{
+		//	var inner = GetVectorInnerImpl();
+		//	if (IsContentMode())
+		//	{
+		//		var vector = make<Vector<IInspectable>>();
+		//		int size = Size();
+		//		for (int i = 0; i < size; i++)
+		//		{
+		//			vector.Append(GetNodeAt(i).Content());
+		//		}
+		//		return vector.GetMany(startIndex, values);
+		//	}
+		//	return inner.GetMany(startIndex, values);
+		//}
+
+		//IVectorView<IInspectable> ViewModel.GetView()
+		//{
+		//	throw hresult_not_implemented();
+		//}
+
+		internal TreeViewNode GetNodeAt(int index)
+		{
+			//	var inner = GetVectorInnerImpl();
+			//	return inner.GetAt(index).as< TreeViewNode > ();
+			throw new NotImplementedException();
+		}
+
+		//void ViewModel.SetAt(uint32_t index, IInspectable const& value)
+		//{
+		//	var inner = GetVectorInnerImpl();
+		//	var current = inner.GetAt(index).as< TreeViewNode > ();
+		//	inner.SetAt(index, value);
+
+		//	TreeViewNode newNode = value.as< TreeViewNode > ();
+
+		//	var tvnCurrent = get_self<TreeViewNode>(current);
+		//	tvnCurrent.ChildrenChanged(m_collectionChangedEventTokenVector[index]);
+		//	tvnCurrent.RemoveExpandedChanged(m_IsExpandedChangedEventTokenVector[index]);
+
+		//	// Hook up events and replace tokens
+		//	var tvnNewNode = get_self<TreeViewNode>(newNode);
+		//	m_collectionChangedEventTokenVector[index] = tvnNewNode.ChildrenChanged({ this, &ViewModel.TreeViewNodeVectorChanged });
+		//	m_IsExpandedChangedEventTokenVector[index] = tvnNewNode.AddExpandedChanged({ this, &ViewModel.TreeViewNodePropertyChanged });
+		//}
+
+		//void ViewModel.InsertAt(uint32_t index, IInspectable const& value)
+		//{
+		//	GetVectorInnerImpl().InsertAt(index, value);
+		//	TreeViewNode newNode = value.as< TreeViewNode > ();
+
+		//	//Hook up events and save tokens
+		//	var tvnNewNode = get_self<TreeViewNode>(newNode);
+		//	m_collectionChangedEventTokenVector.insert(m_collectionChangedEventTokenVector.begin() + index, tvnNewNode.ChildrenChanged({ this, &ViewModel.TreeViewNodeVectorChanged }));
+		//	m_IsExpandedChangedEventTokenVector.insert(m_IsExpandedChangedEventTokenVector.begin() + index, tvnNewNode.AddExpandedChanged({ this, &ViewModel.TreeViewNodePropertyChanged }));
+		//}
+
+		//void ViewModel.RemoveAt(uint32_t index)
+		//{
+		//	var inner = GetVectorInnerImpl();
+		//	var current = inner.GetAt(index).as< TreeViewNode > ();
+		//	inner.RemoveAt(index);
+
+		//	// Unhook event handlers
+		//	var tvnCurrent = get_self<TreeViewNode>(current);
+		//	tvnCurrent.ChildrenChanged(m_collectionChangedEventTokenVector[index]);
+		//	tvnCurrent.RemoveExpandedChanged(m_IsExpandedChangedEventTokenVector[index]);
+
+		//	// Remove tokens from vectors
+		//	m_collectionChangedEventTokenVector.erase(m_collectionChangedEventTokenVector.begin() + index);
+		//	m_IsExpandedChangedEventTokenVector.erase(m_IsExpandedChangedEventTokenVector.begin() + index);
+		//}
+
+		private void Append(object value)
+		{
+			//	GetVectorInnerImpl().Append(value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)